Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -2078,33 +2079,36 @@ public Expression resolveIdentifier(String identifier, boolean mustResolve) {
         // 10: The name of a property on a specific context
         // 11: An unresolved identifier error is thrown
 
+        //Collect every match, issue warning if multiple matches were made, but return initial match:
+        List<Expression> matchCollection = new ArrayList<>();
+
```

Comment:
The warning should provide context about the identifiers, so it would be a List<Pair<Expression, String>>, where expression is the resolved result and String is a description of the resolution.